Mh medic 2.0

bleehu opened the pull request:
Converting Medic to the 2.0 system. 
On reflection, another pass through medic should be made to make it more polished.

bleehu committed:
added armor peircing levels to medic attacks

bleehu committed:
Balanced Armor piercing levels for medic proceedures. Also spell checked.

bleehu committed:
Changed to-miss percentages to conform with Combat 2.0 standards

bleehu committed:
Rounded half accuracy modifiers

1sourcecontrol commented on play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
 Needle Cost:	1
```
line 37, I think "Harmasist" should be "Harmacist", correct me if i'm wrong.

1sourcecontrol commented on play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		2 chance to miss. Deal 52 piercing damage. AP level 3.
```
Were you going to remove the "piercing" part of the damage from the medic abilities? I think it will help keep the players from being confused.

1sourcecontrol reviewed (changes requested):
Just two comments.. Looks great! I see that you nerfed a lot of your abilities, I haven't done any balance checks myself, so I'll be trusting your judgement there.

bleehu commented:
Will make the changes. 
I might need to do another polishing pass to check their balance overall. A lot of the game has changed since I made this class...

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

 Conflicts:
	CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

bleehu committed:
added flavor note to free running.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -50,7 +50,7 @@ Adrenaline Cost:30
-Effect:		16 % chance to miss. Deal 52 damage.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3
 If target has no shield left and is still alive, apply a 
 poison payload of your choice.
```
Fast Spike is purely better than Harmacist Spike, is that intentional?

Also, why does the text say 'no shield left' vs 'if pierces armor'? 
There's a couple of questions this brings... Normally conventional weapons are not stopped by most shields, so why does this reference shields when it isn't blocked by shields?
And is your intention that the needles will always apply poison regardless of piercing armor or not (because that's how it reads to me)?

bleehu replied:
It is an intentional improvement. I should probably have Harmacist spike improve at level 5 to offer a difference in kind.

It is my intention that if the needle does not defeat armor, that no poison is delivered. I will adjust the language.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -60,17 +60,18 @@ Needle Cost: 3
-hands. Each has an 18 % chance to miss. You may pick up to three targets to 
-launch at. Each spike that hits deals 52 damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+hands. Each has an 2 chance to miss. You may pick up to three targets to 
```
>Each has an 2 chance to miss.

an -> a

bleehu replied:
fixed in subsequent commit

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -60,17 +60,18 @@ Needle Cost: 3
-Effect:		10 % chance to miss. Deal 42 damage.
-ignores shields and ballistic armor. Apply a poison payload of your choice.
+Effect:		2 chance to miss. Deal 42 piercing damage. AP level 6.
+if the  needle defeats the target's armor, apply a poison payload of your choice.
```
this doesn't say 'If target has no shield left," is that an intentional difference?
And it should say "pierces the target's armor" or "damages the targets health" because 'defeating' can be ambiguous: do you mean pierce the armor or go all the way through the armor points pool so that you damage health?

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -250,7 +251,8 @@ Duration: 	almost instantaneous
 Effect:		A biologic target afflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite 
-spells for 10 turns if they survive.
+spells for 10 turns if they survive. (Also prevents engineering processes and 
```
Should you change 

>nanite spells

to "Nanite Abilities"? Since we are moving away from the DnD 'spells' aesthetic

1sourcecontrol reviewed (changes requested):
In #228 you mentioned that you would catch the 'nanite' terms in this PR, but I don't see any changes? (For reference, https://github.com/bleehu/Compound_X/blob/95bf27f3ae4d36029353dc83c4a39cb877352280/CharacterCreation/Class-Specific%20Documentation/Medic%20Proceedures.txt#L267)

bleehu committed:
Removed unused Nanite Push Proceedure

bleehu committed:
grammar check, scaled Harmacist spike and fixed shield penetration language

bleehu committed:
stylized Medic Proceedures, added payload clarification and fixed some language on defeating armor

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -564,36 +567,36 @@ Every target revealed by Sense Heartbeat that is communicated to an ally and is
 Range:			Touch
 Action:			Free Action
 Effect:			Transfer n number of Adrenaline points from the Doc to the 
 patient. If This trick is used on another doctor, no adrenaline points are earned. 
```
>Transfer n number of Adrenaline points from the Doc to the 
  patient. If This trick is used on another doctor, no adrenaline points are earned. 

The name doesn't line up with the effect, anymore. Nanite Restore looks more like "Adrenaline Restore," and it doesn't make any sense to transfer adrenaline points to the patient if the patient isn't a doctor.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
-Effect:		10 % chance to miss. Deal 52 damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3. If the spike deals
```
>Deal 52 damage.

Should this say "piercing damage"?

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
-Effect:		16 % chance to miss. Deal 52 damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3. If the spike deals 
```
>Deal 52 damage.

Should this say "piercing damage"?

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -243,16 +250,17 @@ exposed biology are affected by this even if a deployment method says they
 Duration: 	almost instantaneous
 Range:		Payload (use in combination with other powers)
 Effect:		A biologic target afflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite 
```
I missed mentioning another thing---

>prevents them from casting nanite 

Should probably say something like "prevent them from _using_ Nanite"...
Since we don't 'cast ablities/spells' anymore.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
-launch at. Each spike that hits deals 52 damage. If target has no 
-shield left and is still alive, apply a poison payload of your choice.
+hands. Each has a 2 chance to miss. You may pick up to three targets to 
+launch at. Each spike that hits deals 52 piercing damage and has an armor piercing 
```
armor piercing -> Armor Piercing
or -> AP

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -19,60 +19,67 @@ The net effect is that a medic starts every engagement with 100 adrenaline
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 damage. AP level 3. If the spike deals
+damage to the target's health and the target is still alive, apply a poison 
+payload of your choice. At Player level 5, the Harmacist spike deals 64 damage 
```
piercing damage?

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt:
```diff
@@ -264,32 +272,26 @@ recursive without limit, but a target that has made a save against the
 weight. The procedure for doing this without blowing up one's heart or leaving
-one's spine behind takes more training than the average soldier receives. 
+one's spine behind takes more training than the average soldier receives. It 
+also can be quite exhausting.
```
Free Running is so cool! Makes me want to play medic, just to get this ability.

1sourcecontrol reviewed (changes requested):
I'm trying to stay ahead of the game! Good work so far!

bleehu commented:
Thanks for the encouraging comments. Feels like you tore me a new one. But it was necessary. As it is, Medic is off of its aesthetic, and the hodge-podge of old and new wasn't given the level of detail that it needed. I'll handle Issue #234 in this pull as well. It's time to fix my stuff as they say! :)

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

1sourcecontrol commented:
Sure thing! I'm trying to mention my good thoughts, more. One of the down sides to our online source control, it can feel really cold if we only list what's essential for a fix -without giving our thoughts on what we do like.
I think you're doing a great job fixing this up, and making the class in the first place. There's an incredible amount of content in the engineer and doctor/medic classes.

bleehu committed:
spell procedures like the dictionary does

bleehu committed:
updated intro paragraph

bleehu committed:
Medic isn't a class. Renamed Document

bleehu committed:
It's not called Medic, it's called doctor. Also Changed "Quarentine" to "Contaminate Area".
Added some style changes.
 Changes to be committed:
	renamed:    Medic Skill Tree - Hippocratic.png -> Doctor Procedure Tree - Hippocratic.png
	new file:   Doctor Procedure Tree - Malpractice.png
	modified:   Doctor Procedures.txt
	deleted:    Medic Skill Tree - Malpractice.png

bleehu committed:
finished stylizing the Proceedures Doc

bleehu commented:
Made a bunch of changes. Still need to replace
[ ] Nanite Restore
and add
[ ] whateverectomy
There may yet be more issues, but I need to catch a train.

bleehu committed:
added whateverectomy

bleehu committed:
Merge branch 'dev' into MH-medic-2.0

bleehu commented:
Can I get a "fix this moar"? ;)

1sourcecontrol commented:
WEll, I mean, If you really want me to..

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+Fortitude is permanently reduced by 2. You gain 2 Contagions.
+
+    Their Lymph Node: The target loses their lymph node. They lose 40 health and
+120 Nanites. You gain 2 Contagions.
```
>and 120 Nanites

Can we add an "up to" or something? To be crystal clear that the player won't get negative Nanites if they have a pool of 80, for example.

bleehu replied:
Absolutely! Will do! good catch.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+patient over their maximum health.
+
+==Improved P.E.G.s ==
+Adrenaline Reward:	50 Nanites
```
>Adrenaline Reward:	50 Nanites

This isn't technically an Adrenaline reward, is it?

Also, are we strictly keeping Nanites out of doctor "spells" still? (Confirming in order to help me avoid false positives in my debug.)

bleehu replied:
Wow. I can't believe I missed this. Thank you for catching it!

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+Action:			Free Action
+Effect:			Transfer n number of Adrenaline points from the Doc to the 
+patient. If This trick is used on another doctor, no adrenaline points are 
+earned. 
```
Still have "Nanite Restore" here, restoring adrenaline points from doc to x patient.

If the intention is to give the target X Nanites for X adrenaline points (1 for 1) then I'd need some clarification in the description.

bleehu replied:
I need to replace this with something, I'm just not sure what to replace it with.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+area expands to up to 3m x 5m at the discretion of the Doctor.
+
+    "Contaminated" targets get a -2 to melee or weapon accuracy rolls, and lose
+5 health and 5 nanites at the end of their turn every turn until Contamination
```
Idk if we have rules on it, but what happens when the target has 0 Nanites in their Nanite pool? Or what if they had 3 Nanites and need to lose 5?

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+    A biologic target afflicted with this poison must succeed on a DC 110 FORT 
+save or the target takes 180 damage that ignores shields and armor and silences 
+the target. If the target succeeds on their save, they take 60 damage instead. A 
+silenced target is prevented from using nanite abilities for 10 turns. (Also 
```
"nanite" -> "Nanite"

We might want to change the text to say "active abilities that use Nanites" because, strictly speaking, we never formally tag an ability as a "Nanite Ability". Just a thought.

bleehu replied:
Good clarification.

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+Nanite Cost: 15 Nanites
+Duration: 	2 Actions
+Range:		5m radius centered at a point up to 30m from the Doc
+    Your nanites form microscopic bonds with any solid surface allowing 
```
"nanites" -> "Nanites"

FYI I've been capitalizing "nanite/nanites" throughout all of the CX rules. But I'm unsure if we actually do need to capitalize it. I've been treating it like a proper noun, but come to think of it, I don't know if it is. I don't think there is a dictionary definition of "nanite/nanites" yet.

bleehu replied:
Good Call. Will Ctrl+H it.

1sourcecontrol reviewed (changes requested):
Some Nanite'y things

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -0,0 +1,718 @@
+
+==Free Running ==
+Adrenaline Cost: 60 Points
+Nanite Cost: 15 Nanites
```
Can I get your thoughts on this ability? Just curious about the re-introduction of a Nanite Cost.

bleehu replied:
I kinda like the idea of enabling the medic to get to their patients very easily, but I think this ability requires the deployment of nanite technology. Specifically nanite technology over the medic's medical technology. What do you think?

Turtlelord26 replied:
We really need to have that formal design meeting where we hash out once and for all exactly what our resource bar is, how it works, and what position nanites have in the new aesthetic.

1sourcecontrol replied:
Yeah, I guess I'm not sure what to think until we formalize nanite lore. 

My gut says adding Nanites to this specific ability is as good as not having the Nanites cost there at all, if the doctor never uses other nanite abilities. Their pool might be completely unused. Other than that, seems like a good restriction and lore friendly (well, trowl223 era lore friendly).

bleehu committed:
Some grammar and capitolization details cleaned up

1sourcecontrol commented on CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt:
```diff
@@ -325,9 +327,10 @@ Range:		Payload (use in combination with other powers)
-silenced target is prevented from using nanite abilities for 10 turns. (Also 
-prevents Engineering Processes and Doctor Procedures due to violent tremors). At 
-level 13, damage increases to 220.
+silenced target is prevented from using abilities that use Nanites for 10 turns. 
```
We didn't add silenced to the Glossary yet, did we?
Regardless, I think "silences the target" should be changed because the target isn't speaking spells (anymore). Can you rephrase the text to remove or rephrase  'silences'/'silenced'?

Turtlelord26 replied:
"Silence" does sound out of aesthetic, but we should definitely have a word for the state of not being able to use our nanite abilities. How about Suppressed?

1sourcecontrol replied:
"Suppressed" is already taken for being under the effects of Light/Med/Heavy Suppression by a typical ranged weapon. Although I think Suppressed would be a great fit here, as well.

What about "Disrupted"? (Does that have overlap with hacking/robot disabling of some kind?)

Or maybe some term that signifies the inability for the target to calm their arms/physical nerves...?

1sourcecontrol replied:
P.S. I'm not sure if "Suppressed" is added to the glossary yet

Turtlelord26 replied:
Totally forgot about suppression levels :(
How about Stifled?

bleehu replied:
This is the only part of the game (as far as I know) where the Silence effect is used. I don't know that it makes sense to encapsulate it as a status effect? Would other classes/enemies/hazards use a similar Condition?  

If we can make it more aesthetically intuitive by renaming it, we should do that. The reason for the original name of "silenced" comes from the standard industry term for such abilities as used in WoW, LoL, CoH, etc where the term is used to mean "cannot cast spells". Tactically, I think such an ability makes a lot of sense for the medic to have, and I would like to update/polish the ability until it makes sense and works well rather than yanking it. 
Looking forward to more feedback. If I'm understanding this right, this is the last blocker to updating the Doctor class?

bleehu commented:
Mentioning Issue #242 here because merging this fixes it.

bleehu commented:
@Turtlelord26 after 5 rounds of changes requested over a month, I fear this particular pull request has suffered the worst kind of scope creep. If this playable and technically in the combat 2.0 system can we merge it and address any other issues in a more atomic manner?

Turtlelord26 reviewed (approved):
Yes, playable. Non-conflicting with c2.0 systems.
